PanModal/Animator/PanModalAnimator.swift

```diff
@@ -5,6 +5,7 @@
 //  Copyright © 2019 Tiny Speck, Inc. All rights reserved.
 //
 
+#if os(iOS)
```

Comment:
This is needed in all classes to limit it to iOS only? Not able to define in SPM that it's iOS only?

Reply:
Hello. 
No, `#if os(iOS)` needed in only classes using specific iOS library such as `UIKit`. SPM allows to define target platform only for binary, but it doesn't include this property in build source. I found the [article](https://dive.github.io/swift-package-manager/ios/2019/01/20/swift_package_manager_vs_ios.html) with solution for this problem, but I think that isn't good idea.

Reply:
If you use `swift_platform_tools:5.1` you can declare `platform: .iOS(XX)`, this should do the trick

Reply:
Since we now have Catalyst, this should now be:
`#if canImport(UIKit)`. I'm a little late! :(